### ISSUE TITLE
The learning rate plot in Comet is not the expected one

### ISSUE THREAD

**Issue by lucaventurini** [ludwig-ai/ludwig#733]
Hi! I've been trying the comet.ml integration and I must say this has been a great addition to the framework. 🙌

I wanted to exploit it to keep track of the learning rate updates, but the lr being plot is not the one that I expected, especially when trying the learning_rate_warmup_epochs option, which I set to 6 as suggested. The learning rate that is plot on comet is the one set in learning_rate, and it's constant for the first epochs.

Could this be related to this error?

`COMET ERROR: Failed to extract parameters from Optimizer.init()
`

**To Reproduce**
1. Setup comet
2. Set  learning_rate_warmup_epochs option to 6

**Expected behavior**
I expected to see the lr increase in the first 6 epochs, reach the lr set in learning_rate, and eventually decrease, as I set also reduce_learning_rate_on_plateau .

**Actual behavior**
The lr is equal to the set learning_rate in the first epochs, and eventually decreases due to reduce_learning_rate_on_plateau .

**Comment by dsblank** [comment]
@lucaventurini We'd be glad to take a look at that over on https://github.com/comet-ml/issue-tracking/issues

And glad you find it useful!

**Comment by w4nderlust** [comment]
@dsblank let me know if you need help on my side.

**Comment by w4nderlust** [comment]
@lucaventurini  @dsblank where you able to solve the issue?

**Comment by dsblank** [comment]
@Lothiraldan you have some insight into the current status? (I'll be back from vacation next week).

**Comment by dsblank** [comment]
@lucaventurini Did you solve your problem? I didn't see an issue in the comet.ml tracker.

**Comment by w4nderlust** [comment]
@lucaventurini was the problem solved? Can you please try with the latest version on master?
